plugins/github-release-manager/src/cards/patchRc/PatchBody.tsx

```diff
@@ -0,0 +1,301 @@
+          const commitExistsOnReleaseBranch = !!githubDataResponse.value?.recentReleaseBranchCommits.find(
+            ({ sha }) => {
+              return sha === commit.sha;
+            },
+          );
+
+          return (
+            <div style={{ position: 'relative' }} key={`commit-${index}`}>
+              {commitExistsOnReleaseBranch && (
+                <Paper
+                  elevation={3}
+                  style={{
+                    position: 'absolute',
+                    top: '50%',
+                    left: '50%',
+                    transform: 'translate3d(-50%,-50%,0)',
+                    zIndex: 10,
+                    color: 'green',
+                    padding: 6,
+                    background: 'rgba(244,244,244,1)',
+                    borderRadius: 8,
+                  }}
+                >
+                  <FileCopyIcon
+                    fontSize="small"
+                    style={{ verticalAlign: 'middle' }}
+                  />{' '}
+                  Already exists on <b>{releaseBranch?.name}</b>
+                </Paper>
+              )}
+
+              <ListItem
+                disabled={
+                  patchReleaseResponse.loading ||
+                  (patchReleaseResponse.value &&
+                    patchReleaseResponse.value.length > 0) ||
+                  commitExistsOnReleaseBranch
+                }
+                role={undefined}
+                dense
+                button
+                onClick={() => {
+                  if (index === checkedCommitIndex) {
+                    setCheckedCommitIndex(-1);
+                  } else {
+                    setCheckedCommitIndex(index);
+                  }
+                }}
+              >
+                <ListItemIcon>
+                  <Checkbox
+                    edge="start"
+                    checked={checkedCommitIndex === index}
+                    tabIndex={-1}
+                  />
+                </ListItemIcon>
+
+                <ListItemText
+                  id={commit.sha}
+                  primary={commit.commit.message}
+                  secondary={
+                    <>
+                      {commit.sha}{' '}
+                      <Link
+                        color="primary"
+                        href={commit.author.html_url}
+                        target="_blank"
+                      >
+                        @{commit.author.login}
+                      </Link>
+                    </>
+                  }
+                />
+
+                <ListItemSecondaryAction>
+                  <IconButton
+                    aria-label="commit"
+                    disabled={commitExistsOnReleaseBranch}
+                    onClick={() => {
+                      const repoPath = apiClient.getRepoPath();
+
+                      const newTab = window.open(
+                        `https://github.com/${repoPath}/compare/${releaseBranch?.name}...${commit.sha}`,
```

Comment:
This won't work on GHE.  Are you able to perhaps lookup to see if the Backstage config has a hostname for GHE defined and if so use it, then default back to GitHub.com?

Reply:
Good catch!

Addressed in https://github.com/backstage/backstage/pull/5299/commits/c003e06cc004cc085153112af6240a30e7d73200